client/controllers/incidentReports/incidentTable.coffee

```diff
@@ -0,0 +1,33 @@
+Template.incidentTable.onCreated ->
+  @rowsSelected = new ReactiveVar(false)
+  @selectedIncidents = new ReactiveVar([])
+
+changeIncidentStatus = (status, instance) ->
+  $.each instance.selectedIncidents.get(), (index, incidentId) ->
```

Comment:
In other areas of the app we've used either `forEach`, `_.each`, or `for...in`. Maybe we should switch to one of those so we can stay semi-consistent?